### DISCUSSION TITLE
Use of ansible badges

### DISCUSSION THREAD

**Discussion by papanito** [badges/shields#5469] [Q&A] [answered]
For the ansible badges I have to specify a `roleId` which seems numeric. Is that true? Where do I get this `roleId`, I did not find any indication in ansible galaxy

**Comment by calebcartwright** [comment, marked as answer]
https://docs.ansible.com/ansible/latest/cli/ansible-galaxy.html#role-info
https://gist.github.com/pavlov99/c22541856f413920edd95dc61da160e8

**Reply by papanito** [reply]
Thanks!
